projector-plugin/src/main/kotlin/org/jetbrains/projector/plugin/Utils.kt

```diff
@@ -50,6 +51,32 @@ fun productName(): String = ApplicationInfo.getInstance().versionName
 
 fun getPathToPluginDir() = File(PathUtil.getJarPathForClass(ProjectorService::class.java)).parentFile.toString()
 
+private const val VERSIONS_FILE_PATH = "/META-INF/versions.txt"
+private const val AGENT_VERSION = "agentVersion"
+
+fun getAgentVersion(): String {
+  val content = loadVersionsContent()
+  val map = parseVersions(content)
+  val result = map[AGENT_VERSION]
+  require(result != null) { "Can't get agent version" }
```

Comment:
Let's make the error more clear:

```suggestion
  require(result != null) { "Can't get agent version from $VERSIONS_FILE_PATH: $map" }
```